docs/index.rst

```diff
@@ -5,18 +5,81 @@
 Welcome to scikit-fda's documentation!
 ======================================
 
+This package offers classes, methods and functions to give support to
+Functional Data Analysis in Python. Includes a wide range of utils to work with
+functional data, and its representation, exploratory analysis, or
+preprocessing, among other tasks such as inference, classification, regression
+or clustering of functional data.
+
+In the `project page <https://github.com/GAA-UAM/scikit-fda>`_ hosted by
+Github you can find more information related to the development of the package.
+
+
 .. toctree::
-   :includehidden:
-   :maxdepth: 4
+   :maxdepth: 2
    :caption: Contents:
    :titlesonly:
 
    apilist
+
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+
    auto_examples/index
 
-Indices and tables
-==================
+An exhaustive list of all the contents of the package can be found in the
+:ref:`genindex`.
+
+Installation
+------------
+
+Currently, scikit-fda is available in Python 3.6 and 3.7, regardless of the
+platform. The stable version can be installed via
+`PyPI <https://pypi.org/project/scikit-fda/>`_:
+
+.. code-block:: bash
+
+   pip install scikit-fda
+
+
+It is possible to install the latest version of the package, available in
+the develop branch, by cloning this repository and doing a manual installation.
+
+.. code-block:: bash
+
+   git clone https://github.com/GAA-UAM/scikit-fda.git
+   cd scikit-fda/
+   pip install -r requirements.txt
+   python setup.py install
+
+
+In this type of installation make sure that your default Python version is
+currently supported, or change the python and pip commands by specifying a
+version, such as python3.6.
+
+
+Contributions
+-------------
+
+All contributions are welcome. You can help this project grow in multiple ways,
+from creating an issue, reporting an improvement or a bug, to doing a
+repository fork and creating a pull request to the development branch.
+The people involved at some point in the development of the package can be
+found in the `contributors file
+<https://github.com/GAA-UAM/scikit-fda/blob/develop/THANKS.txt>`_.
+
+Citation
```

Comment:
I would comment this section until the paper is published, to not confuse anyone.

Reply:
Done in ffcb066.